src/ui/page/home/home.tsx

```diff
@@ -5,16 +5,18 @@ import { Explore } from './explore/explore'
 import './home.scss'
 import './i18n/index'
 
-export const Home: FC = () => (
-  <div className="okp4-dataverse-portal-home-page-main">
-    <div className="okp4-dataverse-portal-home-page-block-container community">
-      <Community />
+export const Home: FC = () => {
+  return (
```

Comment:
I guess the changes on the Home page is due to Error testing.

Reply:
Ah yes, I forgot about this. Should I change it back?

Reply:
Yess no problem, changing it backk :)